src/app/static/src/js/report.js

```diff
@@ -2,7 +2,7 @@ import Vue from 'vue';
 import $ from 'jquery';
 
 import publishSwitch from './components/reports/publishSwitch.vue'
-import runReport from './components/reports/runReport.vue'
+import runReport from './components/reports/runReportInline.vue'
```

Comment:
I renamed this to avoid confusion with the new `runReport.vue` component. Saw on [this PR](https://github.com/vimc/orderly-web/pull/236) it says the simple runner will be removed once the new page is in, but I'd consider whether there's some value to keeping it - it was very convenient when running the data-viz tool reports to just hit `run report` inline on the report page, so I'm thinking it might still be a useful feature for simple reports with no parameters?

Reply:
Ah good point. As you say we can leave it in situ for parameterless reports, along with the link to the full runner for if the user wants to use a git branch

Reply:
Might be even clearer to user `import runReportInline from...`?